grub2-xen.spec

```diff
@@ -8,18 +8,18 @@
 %define _sourcedir %(pwd)
 %endif
 
-%global tarversion 2.02~beta2
+%global tarversion 2.02
 %undefine _missing_build_ids_terminate_build
 
 Name:           grub2-xen
-Version:        2.02beta2
+Version:        2.02
```

Comment:
I think the initial package had a bug here: missing `~` before `beta2`. This means rpm [will consider](https://blog.jasonantman.com/2014/07/how-yum-and-rpm-compare-versions/#how-rpm-compares-version-parts) `2.02beta2` _newer_ than `2.02`.
Solution for this could be naming this version `2.02final` (`final` is sorted after `beta`). Kind of ugly, but the alternative would be using `Epoch`, which is hard to roll back (if we ever want to drop this package in favor of upstream one).